code/modules/reagents/Chemistry-Goon-Medicine.dm

```diff
@@ -74,17 +72,18 @@ datum/reagent/salglu_solution/on_mob_life(var/mob/living/M as mob)
 datum/reagent/synthflesh
 	name = "Synthflesh"
 	id = "synthflesh"
-	description = "100% chance per cycle of healing 1 point each of BRUTE and BURN damage."
+	description = "50% chance per cycle of healing 1 point each of BRUTE and BURN damage."
 	reagent_state = LIQUID
-	color = "#C8A5DC" // rgb: 200, 165, 220
+	color = "#FFEBEB"
 
 datum/reagent/synthflesh/reaction_mob(var/mob/living/M, var/method=TOUCH, var/volume)
 	if(!M) M = holder.my_atom
 	if(method == TOUCH)
-		M.adjustBruteLoss(-(1*volume)*REM)
-		M.adjustFireLoss(-(1*volume)*REM)
-		M << "<span class='notice'>You feel your burns healing and your flesh knitting together!</span>"
-	..()
+		if(prob(50))
+			M.adjustBruteLoss(-(1*volume)*REM)
+			M.adjustFireLoss(-(1*volume)*REM)
+			M << "<span class='notice'>You feel your burns healing and your flesh knitting together!</span>"
+		..()
 	return
 
```

Comment:
Don't get the synthflesh nerfs. People are making it because of time efficiency and ease of use, not because it's inherently better than silver sulfa or styptic (see: old tricord).